src/common/mfu_daos.c

```diff
@@ -409,14 +407,8 @@ static int daos_set_paths(
             rc = 1;
             goto out;
         }
-        int src_rc = daos_parse_path(src_path, src_len,
-                                     &da->src_pool_uuid, &da->src_cont_uuid);
+        int src_rc = daos_parse_path(src_path, src_len, &da->src_pool, &da->src_cont);
         if (src_rc == 0) {
-            if (!daos_uuid_valid(da->src_cont_uuid)) {
-                MFU_LOG(MFU_LOG_ERR, "Source pool requires a source container.");
-                rc = 1;
-                goto out;
-            }
```

Comment:
@daltonbohning I suppose I could make sure there was something passed in but there is no equivalent to check that it is valid per say? Since it could be a label or uuid now?

Reply:
@daltonbohning I think duns_resolve_path will fail within daos_parse_path though if the src_cont string is not valid.

Reply:
> I suppose I could make sure there was something passed in but there is no equivalent to check that it is valid per say? Since it could be a label or uuid now?

I think this ties in to using both src_cont and src_cont_uuid. Maybe we need to check both? I guess the worst case is that we try to connect to the source container, it's NULL, and the user sees an error then.

> I think duns_resolve_path will fail within daos_parse_path though if the src_cont string is not valid.

But what if it just wasn't passed at all? I.e. NULL

Reply:
@daltonbohning I added a check for a valid uuid or a null src cont string here.